cddl_gen/cddl_gen.py

```diff
@@ -378,6 +382,10 @@ def set_value(self, value_generator):
         value = value_generator()
         self.value = value
 
+        if self.type == "OTHER" and self.value.startswith("$"):
+            self.value = self.value.strip("$")
```

Comment:
Should this be an `lstrip` since `$` inside names is valid?

Reply:
@hakonfam You might be confusing this with `replace`. `strip` doesn't remove characters inside string text, just on the ends. By default it does the same thing as `rstrip().lstrip()`

Reply:
Understood, but still it is valid to end a variable with `$` as in `foo$`, wouldn't this still be an issue?